sys/shell/commands/sc_udp.c

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright (C) 2015 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     sys_shell_commands
+ * @{
+ *
+ * @file
+ * @brief       Shell commands for using UDP
+ *
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
+ */
+
+#include <stdio.h>
+#include <string.h>
+
+#include "net/ng_netbase.h"
+#include "net/ng_udp.h"
+#include "net/ng_ipv6.h"
+#include "net/ng_ipv6/addr.h"
+
+#ifdef MODULE_NG_PKTDUMP
+static ng_netreg_entry_t _server = {NULL,
+                                    NG_NETREG_DEMUX_CTX_ALL,
+                                    KERNEL_PID_UNDEF};
+#endif
+
+
+void _send(char *addr, char *port, char *data)
+{
+    uint16_t port;
```

Comment:
propably not (This PR is not tested too well, and as we discussed above will be heavily re-structured anyway...)